contributed_definitions/nyaml/NXcoordinate_system_set.yaml

```diff
@@ -88,138 +88,259 @@ doc: |
+      doc: |
+        Where is the origin of the detector space reference
+        frame located. This is the location of Xd = 0, Yd = 0, Zd = 0.
+        Assume regions-of-interest in this reference frame form a
+        rectangle or cuboid.
+        Edges are interpreted by inspecting the direction of their
+        outer unit normals (which point either parallel or antiparallel)
+        along respective base vector direction of the reference frame.
+      enumeration: [undefined, front_top_left, front_top_right, front_bottom_right, front_bottom_left, back_top_left, back_top_right, back_bottom_right, back_bottom_left]
+    x_alias(NX_CHAR):
+      doc: |
+        Name or alias assigned to the x-axis base vector,
+        e.g. longest edge as some landmark on the detector.
+    x_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing x-axis base vector of
+        the detector space reference frame. We assume the configuration
+        is inspected by looking towards the sample surface from a
+        position that is located behind the detector.
+        Different tools assume that different strategies can be used
+        and are perceived as differently convenient to enter
+        details about coordinate system definitions. In this ELN users
+        have to possibility to fill in what they assume is sufficient to
+        define the coordinate system directions unambiguously.
+        Software which works with this user input needs to offer parsing
+        capabilities which detect conflicting input and warn accordingly.
+      enumeration: [undefined, north, east, south, west, in, out]
+    y_alias(NX_CHAR):
+      doc: |
+        Name or alias assigned to the x-axis base vector,
+        e.g. long edge as some landmark on the detector.
+    y_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing y-axis base vector of
+        the detector space reference frame. We assume the configuration
+        is inspected by looking towards the sample surface from a
+        position that is located behind the detector.
+        For further information consult also the help info for the
+        xaxis_direction field.
+      enumeration: [undefined, north, east, south, west, in, out]
+    z_alias(NX_CHAR):
+      doc: |
+        Name or alias assigned to the x-axis base vector,
+        e.g. short edge as some landmark on the detector.
+    z_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing z-axis base vector of
+        the detector space reference frame. We assume the configuration
+        is inspected by looking towards the sample surface from a
+        position that is located behind the detector.
+        For further information consult also the help info for the
+        xaxis_direction field.
+      enumeration: [undefined, north, east, south, west, in, out]
+  # conventions specific for certain methods
+  (NXcoordinate_system_em_ebsd):
```

Comment:
`NXcoordinate_system_em_ebsd` should not be part of the generic `NXcoordinate_system_set`, but rather an instance of `NXcoordinate_system` somewhere in the ebsd description.

Reply:
Okay so maybe indeed dropping the base class candidate NXconventions_em_ebsd was a bad choice or indeed copying it over into the NXem_ebsd base class.

BTW: NXem_ebsd is also one such discussion, strictly speaking the NIAC should not vote to support it if they stick to their argumentation of that base classes should be generic, but then all these required definitions to make in - just only one technique of dozens in EM already - would have to go all the way to the appdef. And that is always my point. You happen to agree with everybody about 99.9% the content and definitions in an appdef, but if one constraints you happen to be unable to agree upon you need to define a new appdef  and this has never been convincing to me and I never found anybody who attempted convincing why it is for them. Almost all of our proposed appdefs are on part in terms of thought and complexity covered to many existent appdefs in NeXus

Reply:
Implemented